docker-compose.yml

```diff
@@ -51,6 +52,7 @@ services:
       - env.d/development/postgresql
     volumes:
       - ./data/media:/data/media
+      - ./data/static:/data/static
```

Comment:
I don't think we want this mount on the production stack as the purpose is also to check that static files are well generated in the container?

Reply:
Indeed !

Reply:
still there?